js_tests/admin/inlines.test.js

```diff
@@ -26,3 +26,18 @@ test('add form', function(assert) {
     addButton.click();
     assert.ok(this.table.find('#first-1').hasClass('row2'));
 });
+
+test('inline hooks', function(assert) {
+    assert.expect(2);
+    var $document = django.jQuery(document);
+    var addButton = this.table.find('.add-row a');
+    $document.on( "formset:added", function(event, $row, formsetName) {
```

Comment:
Why a space after ( on this line and the next?
Can we verify the arguments in this test too? Would be nice if we could get away without adding the selenium test in my opinion